Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.h

```diff
@@ -415,6 +415,8 @@ class UnifiedPDFPlugin final : public PDFPluginBase, public WebCore::GraphicsLay
 
     Vector<WebFoundTextRange::PDFData> findTextMatches(const String&, WebCore::FindOptions) final;
     Vector<WebCore::FloatRect> rectsForTextMatch(WebFoundTextRange::PDFData) final;
+    RefPtr<WebCore::TextIndicator> textIndicatorForTextMatch(WebFoundTextRange::PDFData, WebCore::TextIndicatorPresentationTransition) final;
+    void scrollToRevealTextMatch(WebFoundTextRange::PDFData) final;
```

Comment:
Is there a reason we use pass-by-value instead of const ref or rvalue reference here? (since the struct's size is larger than 8 bytes)

Reply:
No reason – fixed to use a const ref!